CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.7.1]
+
+### Changes
+- Update to CCF 4.0.16 (#188)
```

Comment:
Please include also other PRs merged since the previous release (e.g., https://github.com/microsoft/scitt-ccf-ledger/pull/186)

Reply:
Done, Thanks